tests/PHPStan/Rules/Arrays/data/bug-7913.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+namespace Bug7913;
+
+const X = [];
```

Comment:
I tried wrapping it in a method/function, but using `const` like in the repro example, only works in top level code, it seems